registry/strings/helpers.go

```diff
@@ -93,17 +93,6 @@ func (sr *StringsRegistry) initials(str string, delimiters string) string {
 //
 //	string - the transformed string.
 //
-// Example:
-//
-//	style := caseStyle{
-//	    Separator:       '_',
-//	    CapitalizeNext:  true,
-//	    ForceLowercase:  false,
-//	    InsertSeparator: true,
-//	}
-//	transformed := sr.transformString(style, "hello world")
-//	Output: "Hello_World"
-//
 // Note:
 //
 //	This example demonstrates how to use the function to capitalize the first letter of
```

Comment:
You cannot leave this, after removing the example. Please check others functions